### PR TITLE
[GPII-3865]: Refactor organization permission management, second try

### PR DESCRIPTION
**Pull request by natarajaya**
This PR contains changes from #366 plus couple additional fixes:
* It adds missing `organizationAdmin` role to the common SA (required to remove IAM policy bindings)
* It separates `:set_billing_org_perms` task, with updated comments and variable piping

### CODE REVIEW

**Comment by stepanstipl** [comment]
lgtm

File: shared/rakefiles/entrypoint_common.rake [hunk 1]
```diff
@@ -7,12 +7,25 @@ task :apply_common_infra => [:set_vars, :configure_aws_restore] do
...
+end
+
+desc "[ONLY ADMIN] Set required permissions for the accounts in the organization that owns billing account"
+task :set_billing_org_perms => [:set_vars] do
+  # We need to grant Billing User role in the organization that owns
+  # Billing Account to the projectowner SA of current organization, otherwise it will
+  # fail to create new projects.
+  #
+  # WARNING: You need to run this code authenticated under your main (member of cloud-admin group) account.
+  #          It will not work in CI or if you are using projectowner SA.
```
Comment (mrtyler) [inline shared/rakefiles/entrypoint_common.rake]: If this is true, how will we run this code regularly and automatically?

Comment (natarajaya) [reply shared/rakefiles/entrypoint_common.rake]: There is no way to run this code automatically. I updated ONE-TIME-SETUP doc with instructions to add Billing Account User permissions to common SA.

**Comment by natarajaya** [comment]
> You didn't specify, but I assumed the xk_infra.rake changes are the same as the last PR

@mrtyler Your assumption is correct.

File: ONE-TIME-SETUP.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ This document describes manual steps needed for initial configuration when using
 * https://support.google.com/code/contact/billing_quota_increase
    * @mrtyler requested a quota bump to 100 Projects.
       * He only authorized his own email for now, to see what it did. But it's possible other Ops team members will need to go through this step.
+* When configuring additional organizations (i.e. test1.gpii.net), it is important to grant common SA of target organization Billing Account User permissions in the organization that owns main billing account. This can be done by running `rake :set_billing_org_perms`, authenticated as member of cloud-admin group.
```
Comment (mrtyler) [inline ONE-TIME-SETUP.md]: No : on target name.

Comment (mrtyler) [reply ONE-TIME-SETUP.md]: Where does the user run this command?

Comment (natarajaya) [reply ONE-TIME-SETUP.md]: Thanks for the suggestions, updated!

**Comment by natarajaya** [comment]
> You didn't remove the :

@mrtyler Ouch: https://github.com/gpii-ops/gpii-infra/pull/384

> Wait, this is not the PR. Where is this comment?

What do you mean?

**Comment by mrtyler** [comment]
The comment was attached to a commit in the PR -- check out  the header on my previous comment. Anyway it looks like it worked fine. I hadn't seen that particular UI before.

**Pull request merged**